src/hooks/useIsDesktopByMedia.ts

```diff
@@ -0,0 +1,5 @@
+import { useMedia } from 'react-use'
+
+import { breakpoints } from '../utils/theme'
+
+export const useIsDesktopByMedia = () => useMedia(`(min-width: ${breakpoints.l})`)
```

Comment:
```suggestion
export const useIsDesktop = () => useMedia(`(min-width: ${breakpoints.l})`)
```

I find this name odd, maybe `useIsDesktop`, I don't think it's important to know it uses `useMedia`.